### ISSUE TITLE
startService shouldn't be a factory method in the finger tutorial

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3376]
|<img alt="helo's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| helo reported|
|-|-|
|Trac ID|trac#3376|
|Type|defect|
|Created|2008-07-30 18:26:49Z|

f.startService is useless and misleading, according to exarkun ;)

See [doc/core/howto/tutorial/listings/finger/finger15.tac#L50](doc/core/howto/tutorial/listings/finger/finger15.tac#L50)

Attachments:

* [finger15.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/5df/5dfa738b6c92518d508ce804980db107acefd5a2/6a89bcb2d90dd972b5786d8fdfb98ebbbd44e9c1.patch) (493 bytes) - added by cary on 2009-04-02 16:33:43Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__3376 3376
type__defect defect
reporter__helo helo
priority__lowest lowest
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1217442409000000 1217442409000000
changetime__1239650162000000 1239650162000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="cary's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|cary set owner to cary|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="cary's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|cary removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="cary's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|cary set owner to @dreid|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="cary's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|cary removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_6"></a>|@glyph set owner to @glyph|
|-|-|

Looks good.  My goodness these are hard to test.  Merging...

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_7"></a>|@glyph set status to `closed`|
|-|-|

(In [26691]) Remove useless, redundant, misleading `startService` attribute from tutorial.

Author: cary

Reviewer: glyph

Fixes [#3376](https://github.com/twisted/twisted/issues/3376)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
